src/components/registration/Registration.tsx

```diff
@@ -197,7 +197,11 @@ export const Registration: React.FunctionComponent<RegistrationProps> = ({
                 value={state.phone.value}
                 placeholder="Phone #"
                 aria-label="Phone #"
-                onChange={handleOnChange}
+                onChange={(event: React.ChangeEvent<{ value: unknown }>) => {
+                  const value = event.currentTarget.value as any
+                  const isValidPhoneNumber = /^(0|[0-9]\d*)$/.test(value)
```

Comment:
Can we make this func reusable and exported from a commonn place ?
maybe types file for now and then we move all constants to a new file name constants under common directory